src/music_gui.cpp

```diff
@@ -191,14 +192,25 @@ void MusicSystem::SetPositionBySetIndex(uint set_index)
 	if (it != std::end(this->active_playlist)) this->playlist_position = std::distance(std::begin(this->active_playlist), it);
 }
 
+/**
+ * Get set index from current playlist position.
+ * @return current set index, or UINT_MAX if nothing is selected.
+ */
+uint MusicSystem::GetSetIndex()
+{
+	return static_cast<size_t>(this->playlist_position) < this->active_playlist.size()
+		? this->active_playlist[this->playlist_position].set_index
+		: UINT_MAX;
```

Comment:
```suggestion
	if (static_cast<size_t>(this->playlist_position) < this->active_playlist.size()) return this->active_playlist[this->playlist_position].set_index;	
	
	return UINT_MAX;
```
Using a ternary here seems unnecessarily hard to read.

Reply:
Does it? I ... actually think your suggestion is harder to read.

Reply:
It doesn't really matter to me. 🤷